.travis.yml

```diff
@@ -1,26 +1,28 @@
 language: node_js
 node_js: '10'
 cache:
+  yarn: true
   directories:
-    - ~/.npm
+    - node_modules
+    - ~/.cache
 env:
   global:
     - FORCE_COLOR=1
 install:
-  - npm ci
+  - yarn
 jobs:
   include:
     - stage: test
       script:
-        - npm run prettier
-        - npm run tslint
-        - npm run build
-        - npm test -- --single-run --browsers ChromeHeadlessNoSandbox
+        - yarn run prettier
+        - yarn run tslint
+        - yarn run build
+        - yarn test -- --single-run --browsers ChromeHeadlessNoSandbox
```

Comment:
Are the `--` correct for yarn?

Reply:
It's accepted, but deprecated, so I removed `--`.